packages/cms-lib/lib/config.js

```diff
@@ -108,20 +138,23 @@ const getOrderedPortal = unorderedPortal => {
 
 const getOrderedConfig = unorderedConfig => {
   const {
+    defaultAccount,
     defaultPortal,
     defaultMode,
     httpTimeout,
     allowsUsageTracking,
+    accounts,
     portals,
     ...rest
   } = unorderedConfig;
 
   return {
-    defaultPortal,
+    ...(defaultAccount && { defaultAccount }),
+    ...(defaultPortal && { defaultPortal }),
```

Comment:
My thought is we don't specifically disallow having both.  We just favor defaultAccount/accountId over defaultPortal/portalId

Reply:
I think when it comes to writing the file when a user interacts with the CLI (e.g. `hs auth`) maybe we can be more opinionated so that there is only one.

Reply:
That makes sense, though my concern would be if we overwrite an old config with the new key, it could potentially break something on their end.

E.g. if they re-auth something already in their config, it would swap their portalId to accountId